app/views/notes/show.html.erb

```diff
@@ -87,6 +86,24 @@
     </table>
   <% end %>
 
+  <% if current_user && (current_user.role == "admin" || current_user.role == "moderator" || current_user.id == @node.id || current_user.is_coauthor?(@node)) && @node.has_tag('event') %>
+  <hr/>
+  <p>This looks like an event.     <span><button id="date-toggle" class="btn-outline-secondary btn-sm" data-toggle="tooltip" data-placement="right" title="Add Date"><i class="fa fa-calendar" aria-hidden="true"></i></button></span></p>
+  <div id="event-datepicker">
+    <input type="text" class="datepicker" aria-describedby="DatePickerGuide"/>
+
+    <span><button id="save-event-date" class="btn-outline-secondary btn-sm">Save Date</button></span>
+    <small id="DatePickerGuide" class="form-text text-muted">
+      Add date to your awesome event in this box
+    </small>
+
+    <!-- <div class="toast" role="alert" aria-live="assertive" aria-atomic="true">
```

Comment:
Hmm. Regarding hiding it initially, I played around with adding the CSS classes `show` and `fade` and found that `'toast fade show'` is visible but removing `show` makes it invisible. Could you test that out? Maybe we start it without `show` and it would then be hidden by default, and then if we try adding `show` it appears? 

Otherwise you could add `style="display:none;"` and then just use jQuery's `$('.my-toast').show()`. Could that work?

Also would you be able to share a screenshot once you get it working? Thanks a lot, I appreciate your work on this!!!